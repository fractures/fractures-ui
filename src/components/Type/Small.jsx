import PropTypes from "prop-types"
import styled from "styled-components"

const propTypes = {
	children: PropTypes.node.isRequired,
	className: PropTypes.string
}

const Small = styled.small`
	font-size: 0.75rem;
	line-height: 1.25rem;
`

Small.propTypes = propTypes

export default Small
