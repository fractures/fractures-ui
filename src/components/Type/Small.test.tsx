import Small from './Small'

describe('Small', () => {
	const tree = renderer.create(<Small>Lorem ipsum</Small>).toJSON()

	it('Matches snapshot', () => {
		expect(tree).toMatchSnapshot()
	})
})
