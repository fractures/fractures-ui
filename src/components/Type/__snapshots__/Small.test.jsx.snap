// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Small Matches snapshot 1`] = `
<small
  className="sc-bdVaJa iLNvYn"
>
  Lorem ipsum
</small>
`;
